### PR TITLE
Update raycast-system-monitor extension

### PR DESCRIPTION
**Pull request by pernielsentikaer**
## Checklist

- [ ] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ ] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [ ] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ ] I checked that files in the `assets` folder are used by the extension itself
- [ ] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/system-monitor/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # System Monitor Changelog
 
+## [New Action] - 2023-05-31
+
+- Addded Open Activity Monitor action
```
Comment (madza91) [inline extensions/system-monitor/CHANGELOG.md:5]: Typo in "Addded" 🥲

**Review by madza91 — CHANGES_REQUESTED** [review]

File: extensions/system-monitor/CHANGELOG.md (hunk 1 above)
Comment (pernielsentikaer) [reply extensions/system-monitor/CHANGELOG.md]: Good spotted 🔥

**Review by lucasmezencio — APPROVED** [review]
LGTM.

**Review by madza91 — APPROVED** [review]

**Comment by pernielsentikaer** [comment]
Can you approve @hossammourad 🙂

**Review by hossammourad — APPROVED** [review]

**Pull request merged**